### DISCUSSION TITLE
Repository of a project implemented as Orchard decoupled, using Razor Pages and localization of content and pages.

### DISCUSSION THREAD

**Discussion by LjubenIlioski** [OrchardCMS/OrchardCore#15554] [Q&A]
Hi :) I have a new project that want to implement with Orchard. Is there somewhere a repository of a project implemented as decoupled, with Razor Pages, and multi-languages :) I like to check the concept and how complex it will be.  I will be happy if you can share something like that with me. Thanks :)

**Comment by hishamco** [comment]
I'm not sure if there's a repo, I like and use both Razor Pages & Localization for the projects but there's no public one. I might create a quick one if I have a time

**Comment by hishamco** [comment]
You remind me of an old theme that I planned to start with "Meet The Team"

![Screenshot_20-3-2024_32824_](https://github.com/OrchardCMS/OrchardCore/assets/3237266/97427217-4840-43f6-8c8a-e98d48a42a80)

**Comment by Piedone** [comment]
I recommend you check out the following resources:

- https://docs.orchardcore.net/en/latest/docs/guides/decoupled-cms/
- https://www.youtube.com/watch?v=yWpz8p-oaKg
- https://github.com/psijkof/ModernBusiness.OC.RazorPages
- https://learn.microsoft.com/en-us/aspnet/core/razor-pages/
